frontend/projects/upgrade/src/app/core/feature-flags/store/feature-flags.model.ts

```diff
@@ -55,7 +55,7 @@ export interface EmptyPrivateSegment {
 export const NUMBER_OF_FLAGS = 20;
 
 interface IFeatureFlagsSearchParams {
-  key: FLAG_SEARCH_KEY;
+  key: string;
```

Comment:
i agree that FLAG_SEARCH_KEY is more specific. it is an admission that I failed to get the better type to work 😞. enums in typescript sometimes end up being more work than they are worth i have found, because they often make you have to write ridiculous code to get the types to "work" with barely more type-safety than a string. This was making me have to write ridiculous code in some places, which seemed not very worth it.

Reply:
yes but you kept the enum at a few places, isn't it throwing an error there

Reply:
i'll take another look to see if I can make it work.

Reply:
ok, it may be working ok as the enum now, i'm not sure what the headaches were that it was giving.